bungee/src/main/java/us/myles/ViaVersion/bungee/handlers/BungeeDecodeHandler.java

```diff
@@ -20,6 +20,10 @@ public BungeeDecodeHandler(UserConnection info) {
 
     @Override
     protected void decode(final ChannelHandlerContext ctx, ByteBuf bytebuf, List<Object> out) throws Exception {
+        if (!ctx.channel().isActive()) {
+            return;
```

Comment:
I guess this will throw an exception, using more cpu. Use the stackless cancelexception

Reply:
> I guess this will throw an exception, using more cpu. Use the stackless cancelexception

What do you think about using 'ClosedChannelException'?

Reply:
Fixed in d0941b5